lib/cloudkeeper/one/core_connector.rb

```diff
@@ -1,36 +1,84 @@
 module Cloudkeeper
   module One
     class CoreConnector < CloudkeeperGrpc::Communicator::Service
+      attr_reader :image_handler, :template_handler, :datastore_handler, :group_handler
+
+      include Cloudkeeper::One::ApplianceActions::Registration
+      include Cloudkeeper::One::ApplianceActions::Removal
+      include Cloudkeeper::One::ApplianceActions::Update
+      include Cloudkeeper::One::ApplianceActions::List
+
+      ERRORS = Hash.new(:ERROR).update(Cloudkeeper::One::Errors::Actions::ListingError => :ERROR_APPLIANCE_NOT_FOUND,
```

Comment:
Didn't `rubocop` complain about this? Using a complex unfrozen structure in a constant? I'm asking out of interest, this is not a negative comment :-)